pkg/apis/performance/v1alpha1/performanceprofile_types.go

```diff
@@ -12,7 +12,17 @@ type PerformanceProfileSpec struct {
 	CPU *CPU `json:"cpu,omitempty"`
 	// HugePages defines set of huge pages related parameters.
 	HugePages *HugePages `json:"hugepages,omitempty"`
-	// NodeSelector is a selector which must be true for the performance profile to fit on a node.
+	// MachineConfigLabel defines the label to add to the MachineConfigs the operator creates. It has to be
+	// used in the MachineConfigSelector of the MachineConfigPool which targets this performance profile.
+	MachineConfigLabel map[string]string `json:"machineConfigLabel,omitempty"`
+	// MachineConfigPoolSelector defines the MachineConfigPool label to use in the MachineConfigPoolSelector
+	// of resources like KubeletConfigs created by the operator.
+	// Defaults to "machineconfiguration.openshift.io/role=<same role as in NodeSelector label key>"
+	MachineConfigPoolSelector map[string]string `json:"machineConfigPoolSelector,omitempty"`
+	// NodeSelector defines the Node label to use in the NodeSelectors of resources like Tuned created by the operator.
+	// It most likely should, but does not have to match the node label in the NodeSelector of the MachineConfigPool
+	// which targets this performance profile.
+	// Defaults to "machineconfiguration.openshift.io/role=<same role as in NodeSelector label key>"
```

Comment:
i think this was supposed to go with the MachineConfigLabel, probably a copy and paste error

Reply:
yes, good catch, fixed